mycroft/audio/speech.py

```diff
@@ -35,16 +35,15 @@
 _last_stop_signal = 0
 
 
-def _start_listener(message):
-    """
-        Force Mycroft to start listening (as if 'Hey Mycroft' was spoken)
-    """
-    create_signal('startListening')
+def _start_listener(_):
```

Comment:
Yes it's used from [L66](https://github.com/MycroftAI/mycroft-core/pull/2345/files#diff-1d158d7b163e74b95987f0e65bda74b9R66)

The functionality is somewhat changed however since it emits the mycroft.mic.listen instead of setting the signal file

Reply:
:+1: